### ISSUE TITLE
{Linked:Bug1763121}[Adaptive Cards > Designer] Screen reader does not provide the status message after workspace connected.

### ISSUE THREAD

**Issue by Raisul123** [microsoft/AdaptiveCards#9276]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS version: 25H2(OS Build 26220.6780)
Edge-dev Version: 143.0.3624.0 (Official build) dev (64-bit)

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1.	Open the above URL on the Edge Dev browser
2.	Navigate to the Select a workspace for Copilot Card using tab key.
3.	Choose a workspace from the dropdown list and click “Select and continue.”
4.	Observe and verify that the Screen reader does not provide the status message after workspace connected or not.

**Actual Result:**
Screen reader does not provide the status message after workspace connected.
**Observation:** After connecting to the workspace, a confirmation message appears in green, but the screen reader remains silent and does not announce any information.

**Expected Result:**
After connecting to the workspace, a confirmation message appears in green as "you've connected". Screen reader should provide the status message.

**Issue Description:**
After successfully connecting to a workspace, screen readers announce no confirmation or status message. This missing feedback prevents users relying on assistive technologies from knowing whether the action was completed successfully.

**User Impact:**
Users relying on screen readers, such as individuals with visual impairments, do not receive audible confirmation after successfully connecting to a workspace. This lack of feedback creates uncertainty about whether the action was completed, often causing users to repeat steps unnecessarily or abandon the process entirely. As a result, accessibility is hindered, user trust is compromised, and overall task completion and usability are negatively impacted

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/status-messages

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/c8dd27fe-5eb6-4128-8e72-3c3b1f5c5cd1

### Card JSON

```JSON
{
"actions": null,
"type": "AdaptiveCard",
"body": [
{
"wrap": true,
"type": "TextBlock",
"size": "large",
"weight": "bolder",
"text": "Select a workspace for Copilot"
},
{
"type": "ColumnSet",
"columns": [
{
"type": "Column",
"items": [
{
"type": "TextBlock",
"spacing": "none",
"text": "Choose the workspace where your Copilot usage costs will be billed. Your selection is saved across devices and sessions. This won’t limit your work, but the workspace must meet Copilot requirements. [Learn more](https://go.microsoft.com/fwlink/?linkid=2249079)",
"isSubtle": true,
"wrap": true,
"size": "small"
},
{
"type": "Input.ChoiceSet",
"id": "workspaceChoiceSet",
"choices": [
{
"title": "FirstWorkspace Name",
"value": "4257842578247"
},
{
"title": "SecondWorkspace Name",
"value": "777775757575"
}
],
"placeholder": "Select",
"spacing": "none",
"label": "\r\nWorkspaces",
"style": "filtered"
},
{
"type": "ColumnSet",
"columns": [
{
"type": "Column",
"width": "170px",
"items": [
{
"type": "ActionSet",
"actions": [
{
"type": "Action.Execute",
"title": "Select and continue",
"id": "SELECT_AND_CONTINUE_WS_ID",
"isEnabled": true,
"verb": "SELECT_AND_CONTINUE_WS_ID",
"data": {
"id": "SELECT_AND_CONTINUE_WS_ID"
},
"style": "positive"
}
],
"spacing": "none",
"height": "stretch"
}
]
}
]
}
],
"width": "420px",
"minHeight": "110px"
}
]
}
],
"$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
"version": "1.5",
"minHeight": "226px",
"verticalContentAlignment": "center"
}
```
